script/1_0_2current_en_script/Ciel/Day 2/02_01_CIEL02_1_3.txt

```diff
@@ -73,15 +73,15 @@ For some reason, I specifically avoided asking 'Do you like curry?'
 <Page24>
 Good... At least it looks like she's aware of it.
 <Page25>
-"But I understand. If you don't buy enough of what you like, you'll get uneasy.
+"But I understand. If you don't buy enough of what you like, you get uneasy.
 Now then, I'll take your word for it and help myself to one."
 <Page26>
 I pick up one of the curry bread spread out on Senpai's lap and bring it to my mouth.
```

Comment:
I pick up a curry bread from the selection spread out on Senpai's lap and bring it to my mouth.